runtime/src/accounts.rs

```diff
@@ -787,6 +787,7 @@ impl Accounts {
             "load_to_collect_rent_eagerly_scan_elapsed",
             ancestors,
             range,
+            true,
```

Comment:
This 'true' causes us to find all account indexes that match the range. The closure loads all the accounts and returns (pubkey, AccountSharedData). So, it doesn't matter at all how many accounts are returned per iteration of the deep scan. The 'true' also avoids the sort completely. So, this change is equivalent (at least for rent collection) to eliminating the sort and returning a full bucket of account index items at a time. #19299. But, this change also causes fallback rpc scans (when there is no secondary index) to work as well.